contrib/check-config.sh

```diff
@@ -253,17 +253,11 @@ if [ "$kernelMajor" -lt 3 ] || ( [ "$kernelMajor" -eq 3 ] && [ "$kernelMinor" -l
 	check_flags RESOURCE_COUNTERS
 fi
 
-if [ "$kernelMajor" -lt 3 ] || ( [ "$kernelMajor" -eq 3 ] && [ "$kernelMinor" -le 13 ] ); then
-	netprio=NETPRIO_CGROUP
-else
-	netprio=CGROUP_NET_PRIO
-fi
-
 flags=(
-	BLK_CGROUP BLK_DEV_THROTTLING IOSCHED_CFQ CFQ_GROUP_IOSCHED
+	BLK_CGROUP BLK_DEV_THROTTLING "(IOSCHED_CFQ|IOSCHED_BFQ)" "(CFQ_GROUP_IOSCHED|BFQ_GROUP_IOSCHED)"
 	CGROUP_PERF
 	CGROUP_HUGETLB
-	NET_CLS_CGROUP $netprio
+	NET_CLS_CGROUP "(NETPRIO_CGROUP|CGROUP_NET_PRIO)"
```

Comment:
This is not the same as the old behavior.

Reply:
Not exactly, but it is the method requested in the issue that this PR closes, so unless this breaks something practical it should stay.